core/src/main/java/io/aiven/klaw/service/AnalyticsControllerService.java

```diff
@@ -77,12 +79,12 @@ public String getEnvName(String envId) {
   }
 
   // For Sync Back Acls
-  public Map<String, String> getAclsCountPerEnv(String sourceEnvSelected) {
+  public AclsCountPerEnv getAclsCountPerEnv(String sourceEnvSelected) {
     int tenantId = commonUtilsService.getTenantId(getCurrentUserName());
 
     List<Map<String, String>> aclsPerEnvList =
         manageDatabase.getHandleDbRequests().selectAclsCountByEnv(null, tenantId);
-    Map<String, String> resultMap = new HashMap<>();
+    AclsCountPerEnv resultMap = new AclsCountPerEnv();
 
```

Comment:
I'd say Refactor that name from resultMap to avoid confusion

Reply:
agree, updated all those instance variables